lib/CohereEmbedV3ServicesStackProps.ts

```diff
@@ -52,6 +52,10 @@ export interface CohereEmbedV3Services {
      * The memory type for the deployment.
      */
     readonly cdkDeployMemoryType: string;
+    /**
+     * The vpcId for the deployment.
+     */
+    readonly vpcId: string;
```

Comment:
**Suggestion:** Add a brief description to the `vpcId` property in the interface to improve code documentation.
> 🏰  **High Priority** 
 > [category: **`best practice`**, importance: **`8`**]
> **Reason:** Adding a description to the `vpcId` property improves code documentation and helps other developers understand the purpose of this property.
 

```suggestion
     56 +    /**
     57 +     * The VPC ID for the deployment.
     58 +     */
     59 +    readonly vpcId: string;
```